config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -31,6 +31,41 @@ periodics:
     testgrid-tab-name: capz-periodic-conformance-v1alpha4-main
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
     description: Runs conformance & node conformance tests on a stable k8s version with latest cluster-api-provider-azure
+- name: periodic-cluster-api-provider-azure-upstream-v1alpha4-windows
```

Comment:
can you add a pull optional/don't run by default job to validate PRs that touch the tests themselves

Reply:
maybe in a separate PR so we can merge it first

Reply:
https://github.com/kubernetes/test-infra/pull/21748